build/common.ps1

```diff
@@ -16,6 +16,8 @@ Set-Alias nuget $NuGetExe
 Set-Alias xunit $XunitConsole
 Set-Alias ilmerge $ILMerge
 
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
Prior to this change, 3 separate CI builds failed with the following errors:
```
[15:17:36 +0]	Downloading .NET CLI
##[error]Error-Log : [15:17:37 +0]	[FAILED +00:00:00] Installing .NET CLI
At E:\A\_work\212\s\build\common.ps1:132 char:17
+ ...             Error-Log "[FAILED +$(Format-ElapsedTime $sw.Elapsed)] $B ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Error-Log
 
Invoke-WebRequest : The underlying connection was closed: An unexpected error occurred on a send.
At E:\A\_work\212\s\build\common.ps1:216 char:9
+         Invoke-WebRequest 'https://dot.net/v1/dotnet-install.ps1' -Ou ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc 
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
```
...
```
##[error]E:\A\_work\212\s\configure.ps1 : Build's completed with 10 error(s):
>>> 
>>> The underlying connection was closed: An unexpected error occurred on a send.
>>> The underlying connection was closed: An unexpected error occurred on a send.
>>> 
>>> The underlying connection was closed: An unexpected error occurred on a send.
>>> 
>>> The underlying connection was closed: An unexpected error occurred on a send.
>>> The underlying connection was closed: An unexpected error occurred on a send.
>>> [15:17:37 +0]	[FAILED +00:00:00] Installing .NET CLI
>>> The underlying connection was closed: An unexpected error occurred on a send.
At line:1 char:1
+ . 'E:\A\_work\212\s\configure.ps1' -Force -CleanCache
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,configure.ps1
 

##[error]Process completed with exit code 1 and had 2 error(s) written to the error stream.
```